packages/bpk-stylesheets/webpack.config.babel.js

```diff
@@ -73,7 +73,18 @@ module.exports = {
             loader: 'css-loader',
             options: {
               importLoaders: 1,
-              modules: 'global',
+              modules: {
+                namedExport: true,
+                localIdentName: '[hash:base64]',
+                localIdentHashFunction: 'md4',
+                mode: (resourcePath, resourceQuery, resourceFragment) => {
+                  if (/global.scss$/i.test(resourcePath)) {
+                    return "global";
+                  }
+    
+                  return "local";
+                },
+              },
```

Comment:
Could it be possible of adding some comment here as to what this checking is doing and the config option `mode`